12/Dockerfile.rhel8

```diff
@@ -43,7 +43,7 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
 RUN yum -y module enable postgresql:12 && \
-    INSTALL_PKGS="rsync tar gettext bind-utils nss_wrapper postgresql-server postgresql-contrib" && \
```

Comment:
MR, with the title "Support for building and testing PSQL-16 on C10S," also changes dockerfiles for RHEL8/9. It is a little bit confusing. Maybe I would advise to add the reason for this change to the commit message. Otherwise, I do not see any technical blocker. LGTM

Reply:
replaced with different commit message https://github.com/sclorg/postgresql-container/pull/587/commits/21c1929afd740cd59f50243685c05a26285d2a84